pallets/automation-time/src/lib.rs

```diff
@@ -612,37 +610,34 @@ pub mod pallet {
 		///
 		/// Returns a vec with the tasks that were not run and the remaining weight.
 		/// TODO (ENG-157): calculate weights.
-		fn run_missed_tasks(
+		pub fn run_missed_tasks(
 			mut task_ids: Vec<T::Hash>,
 			mut weight_left: Weight,
 		) -> (Vec<T::Hash>, Weight) {
-			// need to calculate the weight of the fn minus the loop.
-			weight_left -= 10_000;
-
 			let mut consumed_task_index: usize = 0;
 			for task_id in task_ids.iter() {
 				consumed_task_index += 1;
 
 				let action_weight = match Self::get_task(task_id) {
 					None => {
 						Self::deposit_event(Event::TaskNotFound { task_id: task_id.clone() });
-						10_000
+						<T as Config>::WeightInfo::run_missed_tasks_many_missing(1)
 					},
 					Some(task) => {
 						Self::deposit_event(Event::TaskMissed {
 							who: task.owner_id.clone(),
 							task_id: task_id.clone(),
 						});
 						Tasks::<T>::remove(task_id);
-						10_000
+						<T as Config>::WeightInfo::run_missed_tasks_many_found(1)
 					},
 				};
 
 				// need to calculate the look cost minus the action
-				weight_left = weight_left - action_weight - 10_000;
+				weight_left = weight_left - action_weight;
 
 				// need to calculate the max cost of the loop
-				if weight_left < 20_000 {
+				if weight_left < <T as Config>::WeightInfo::run_missed_tasks_many_found(1) {
```

Comment:
just to double check my sanity here, max of loop should just be run_missed_tasks_many_found, right?